widgets/src/c3/Gauge.js

```diff
@@ -0,0 +1,43 @@
+(function (root, factory) {
+    if (typeof define === "function" && define.amd) {
+        define(["./Common"], factory);
+    } else {
+        root.Gauge = factory(root.Common);
+    }
+}(this, function (Common) {
+    function Gauge(target) {
+        Common.call(this);
+
+        this._class = "c3_gauge";
+        this._type = "gauge";
+        this._config.data = {
+            columns: []
+        };
+    };
+    Gauge.prototype = Object.create(Common.prototype);
+
+    Gauge.prototype.palette = function (_) {
+        if (!arguments.length) return this._palette;
+        this._config.color.pattern = _;
```

Comment:
If "_" is one of our Palettes - I don't think this will work. If it isn't then we should rename the method name to c3_palette or such like?